### PR TITLE
[coordenadas-do-cep] New @types for brazillian postal code lib

### PR DESCRIPTION
**Pull request by cybalencar96**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## coordenadas-do-cep ([<kbd>unpkg</kbd>](https://unpkg.com/browse/coordenadas-do-cep@latest/))
was missing the following properties:
1. The declaration doesn't match the JavaScript module 'coordenadas-do-cep'. Reason:
The JavaScript module can be called or constructed, but the declaration module cannot.

The most common way to resolve this error is to use 'export =' syntax.
To learn more about 'export =' syntax, see https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require.
1. busca_aproximada
1. precisao
1. casas_dec_dist
1. round

<details>
<summary>as well as these 2 other properties...</summary>
<p>get_https, radianos</p>
</details>

<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 0390b9150947bbb4e0a90b5dc41b7512ae44d342
</p>

File: types/coordenadas-do-cep/tslint.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{ 
+    "extends": "@definitelytyped/dtslint/dt.json",
+    "rules": {
```
Comment (amcasey) [inline types/coordenadas-do-cep/tslint.json:3]: Please remove the `rules` property and, if there are lint errors, add line-level suppressions.

**Review by amcasey — CHANGES_REQUESTED** [review]
LGTM, other than the linter suppressions.

**Comment by amcasey** [comment]
@cybalencar96 It looks like that particular error has to be disabled in the lint file and your exports look okay, so I'm fine with that.  Sorry for the goose chase.

**Pull request closed without merging**